app/middleware.go

```diff
@@ -74,8 +76,14 @@ func (l LoggerMiddleware) Serve(next echo.HandlerFunc) echo.HandlerFunc {
 
 		startTime = time.Now()
 
+		metricTagMap := make(map[string]interface{})
+		c.Set("metricTagsMap", metricTagMap)
+
 		result := next(c)
 
+		caller := c.Get("metricTagsMap")
+		caller = caller.(map[string]interface{})["gameID"]
```

Comment:
I think we can simplify this:

```suggestion
		caller := metricTagsMap["gameID"]
```

Reply:
Actually, when I use `caller := metricTagsMap["gameID"]` the gameId gets `nil` value instead of the game itself.